integration_test/integration_test.go

```diff
@@ -0,0 +1,322 @@
+/**
```

Comment:
`integration_test` as a top-level directory name is a bit weird - too long, shorter names are more common for top-level directories. I'd name it "testing".

Reply:
Done.